packages/calling-component-bindings/src/utils/videoGalleryUtils.ts

```diff
@@ -73,8 +73,15 @@ export const _videoGalleryRemoteParticipantsMemo: _VideoGalleryRemoteParticipant
             participant.role,
             isHideAttendeeNamesEnabled
           );
-          /* @conditional-compile-remove(reaction) */
-          const remoteParticipantReaction = memoizedConvertToVideoTileReaction(participant.reactionState);
+          let contentSharingStream = undefined;
```

Comment:
Still can't understand why we cannot CC those values, let's CC those unless we have to keep them? Is it because memoizeFn?

Reply:
Yes, it is because memoizeFn is a func with multiple args. For example, if we have the following function:
```
const func = (x: X, /*cc-remove(A)*/ y?: Y, /*cc-remove(B)*/ z?: Z) => {...}

/*cc-remove(A)*/
const y = ...
/*cc-remove(B)*/
const z = ...
func(x, /*cc-remove(A)*/ y, /*cc-remove(B)*/ z)
``` 
we can't stabilize feature B if feature A is not stabilized because all args after `/*cc-remove(A)*/` is removed.

So to get around this, we can define the conditionally compiled argument like so:

```
const func = (x: X, y: undefined | /*cc-remove(A)*/ Y, z: undefined | /*cc-remove(B)*/ Z) => {...}

let y = undefined;
/*cc-remove(A)*/
y = ...
let z = undefined;
/*cc-remove(B)*/
y = ...
func(x, /*cc-remove(A)*/ y, /*cc-remove(B)*/ z)
```

Reply:
Looks like a problem with CC? Will reach out to you to see if there is anything we need to fix for CC

We can keep this as it is in your PR then come back later once we fix it